tests/fixtures/saas/shopify_fixtures.py

```diff
@@ -33,102 +23,163 @@ def shopify_secrets(saas_config):
         "domain": pydash.get(saas_config, "shopify.domain") or secrets["domain"],
         "access_token": pydash.get(saas_config, "shopify.access_token")
         or secrets["access_token"],
+        "fixtures_access_token": pydash.get(
```

Comment:
How is this `fixtures_acess_token` different from the other `access_token`? If it has write access for example, we should rename it to be able to tell how it's different

Reply:
Yes, it has write access for the fixture data operations: creating customers, orders, blogs, articles and comments. 

What would be a good name? I was using `fixtures_access_token` to signal that it was used on the fixtures only.  Other prefixes that i can think is 
*  `write_`
*  `data_creation_`
* `seeding_`

I still think that `fixtures_` is better, since it signals _where_ it's used